core/model/patron.py

```diff
@@ -571,6 +576,30 @@ def until(self, default_loan_period):
         return start + default_loan_period
 
 
+# Finland
+class LoanCheckout(Base, LoanAndHoldMixin):
+    """A model to keep track of loan history, i.e. past checkouts. Similar to `Loan` model with some fields omitted and timestamp added"""
+
+    __tablename__ = "loancheckouts"
+    id = Column(Integer, primary_key=True)
+
+    patron_id = Column(
+        Integer, ForeignKey("patrons.id", ondelete="CASCADE"), index=True
+    )
+    patron: Patron
+
+    license_pool_id = Column(Integer, ForeignKey("licensepools.id"), index=True)
+    license_pool: Mapped[LicensePool] = relationship("LicensePool")
+
+    license_id = Column(Integer, ForeignKey("licenses.id"), index=True, nullable=True)
```

Comment:
Is license_id needed?

Reply:
@ttuovinen or @jompu might be able to answer.